src/revlanguage/distributions/math/Dist_logExponential.cpp

```diff
@@ -40,13 +39,9 @@ Dist_logExponential* Dist_logExponential::clone( void ) const
 }
 
 
-RevBayesCore::LogExponentialDistribution* Dist_logExponential::createDistribution( void ) const
+RevBayesCore::ContinuousDistribution* Dist_logExponential::createDistribution( void ) const
 {
-    // get the parameters
-    RevBayesCore::TypedDagNode<double>*       l = static_cast<const Real &>( lambda->getRevObject() ).getDagNode();
-    RevBayesCore::LogExponentialDistribution* d = new RevBayesCore::LogExponentialDistribution( l );
-    
-    return d;
+    throw RbException()<<"dnLogExponential has been removed from RevBayes.\n  * If you want a distribution that is uniform on orders of magnitude, consider dnLoguniform.\n  * If you want a distribution whose log is exponential, consider dnLog(dnExponential(lambda))";
```

Comment:
Why did you remove the distribution? This breaks older scripts that used this distribution. One option could be to internally create LogDistribution if you want to remove some c++ files. The same should apply to the LogUniform distribution?

Reply:
My reasoning is that there is no really good option here. I think dnLogExponential had the wrong density before.  At first I fixed the density, but that silently changes the *meaning* of the older scripts, while allowing them to run.  For example, previously `x ~ dnLogExponential(1)` could return a negative `x`.  However, when the correct density is used, `x` is always positive.  So it seems like the previous scripts need to be modify somehow.  This option is not great because it breaks the older scripts -- but at least it tells the user how to fix them.

I don't like breaking the older scripts, but I also worry about changing their meaning silently.  I see that dnLogExponential is used only in one tutorial that was written by Mike May in 2022.  We could ask him what he wants to do...